tests/test_insert.py

```diff
@@ -7,13 +8,19 @@ def test_insert_into():
 
 def test_insert_into_with_columns():
     assert_table_lineage_equal(
-        "INSERT INTO tab1 (col1, col2) SELECT * FROM tab2;", {"tab2"}, {"tab1"}
+        "INSERT INTO tab1 (col1, col2) SELECT * FROM tab2;",
+        {"tab2"},
+        {"tab1"},
+        skip_graph_validation=True,
```

Comment:
Had to skip validation for graph as now we are adding the target columns into graph as edge which is not happing for sqlparse yet

Reply:
Since we're not adding sqlparse support for this feature, I'll just mark test_sqlparse=False and add some comment so we know table-level lineage actually works for this case.